custom/ewsghana/reports/specific_reports/reporting_rates.py

```diff
@@ -36,7 +32,9 @@ class ReportingRates(EWSData):
     def rows(self):
         rows = {}
         if self.config['location_id']:
-            supply_points = get_supply_points(self.config['location_id'], self.config['domain'])
+            supply_points = get_supply_points(self.config['location_id'], self.config['domain']).values_list(
+                *['supply_point_id'], flat=True
```

Comment:
is there a reason you use `values_list(*['supply_point_id'], ...)` instead of the simpler `values_list('supply_point_id', ...)`? (applies to many places here and also in https://github.com/dimagi/logistics/pull/189)